source/beamline/BioXAS/BioXASSideTableCoordinator.cpp

```diff
@@ -388,8 +371,11 @@ void BioXASSideTableCoordinator::manipulateVerticalPVChange(double upstreamHeigh
 
 void BioXASSideTableCoordinator::manipulateHorizontalPVChange(double upstreamOffset, double downstreamOffset, AMControl* lateralPV, AMControl* yawPV)
 {
-	double lateral = ( downstreamOffset + upstreamOffset ) / 2;
-	double yaw = radianToDegree(atan((downstreamOffset - upstreamOffset) / (tableHorizontalMotorPosition_ * 2)));
+	double lateral = calculateTableLateral(upstreamOffset, downstreamOffset);
+	double yaw = calculateTableYaw(upstreamOffset, downstreamOffset);
+
+	lateral = qRound(lateral*1000)/1000.0;
```

Comment:
@dretrex I got what you mean. I might should use `PREC` in the DB file, then I don't need to round here anymore.